Mage/src/main/java/mage/abilities/effects/common/continuous/GainControlTargetEffect.java

```diff
@@ -105,8 +105,7 @@ public boolean apply(Game game, Ability source) {
                     controlChanged = true;
                 }
             }
-            if (source.isActivatedAbility()
-                    && firstControlChange && !controlChanged) {
+            if (firstControlChange && !controlChanged) {
```

Comment:
Old code was used for GUI messages fix from #5922. Make sure it works fine in real game (unit tests can't catch such restricted messages).

Reply:
I just tried the scenario from that issue in a real game, and everything worked fine. No message spam or other issues.